### PR TITLE
v0.1.0

### PR DESCRIPTION
**Pull request by notgull**
- Initial commit

### CODE REVIEW

File: Cargo.toml [hunk 1]
```diff
@@ -1,8 +1,16 @@
 [package]
 name = "fastrand-contrib"
+# Make sure to modify CHANGELOG.md before releasing a new version.
 version = "0.1.0"
+authors = ["John Nunley <dev@notgull.net>", "Petr Nevyhoštěný <petr.nevyhosteny@gmail.com>"]
```
Comment (notgull) [inline Cargo.toml:5]: @pnevyk Used your email from your git commits here, hope that's alright

Comment (pnevyk) [reply Cargo.toml:5]: Yeah, it's fine. Thanks for letting me know.

**Comment by pnevyk** [comment]
@notgull I added a commit to this branch which copies the documentation of `libm` feature from lib.rs to readme, because I realized that it's missing there after I merged the normal sampling PR. I hope it's fine.

File: Cargo.toml [hunk 2]
```diff
@@ -1,8 +1,16 @@
...
 edition = "2018"
 rust-version = "1.43.1"
```
Comment (taiki-e) [inline Cargo.toml]:
```suggestion
rust-version = "1.43"
```

`rust-version = "1.43.1"` rejects build on 1.43.0, but there is no API that available on 1.43.1 but not on 1.43.0.

**Comment by notgull** [comment]
> @notgull I added a commit to this branch which copies the documentation of `libm` feature from lib.rs to readme, because I realized that it's missing there after I merged the normal sampling PR. I hope it's fine.

Thanks! In the future could you open a PR for that (or at least push to `main`) instead of pushing to the new version PR? I like to keep it so that the `vX.Y.Z` PR only has the commit that bumps the version.

**Comment by pnevyk** [comment]
@notgull Ah sorry, I didn't realize it's a "special" PR. You could definitely discard that commit from here and I would create a separate PR. I will keep that in mind.

**Comment by notgull** [comment]
It's all good, I've already merged it into `main` anyways.

**Pull request merged**